src/contracts/proxy-deposit-abi.json

```diff
@@ -2,11 +2,11 @@
   "networks": {
     "1": {
       "links": {},
-      "address": "0x029bB89d64695D6A461eEbC1Aab4a4C8657a3f22"
```

Comment:
Why are you filling this in for mainnet as well? Doesn't seem to be any contract deployed at this address on mainnet: https://etherscan.io/address/0x174FBe9Bf9B8838c4DfA875c29368A17e8877E6A

Reply:
its just a placeholder for now